core/templates/pages/library-page/library-page.component.html

```diff
@@ -38,8 +38,7 @@ <h2 class="oppia-group-page-header">
       </div>
 
       <div class="oppia-exp-summary-tiles-container" ng-if="$ctrl.pageMode === $ctrl.LIBRARY_PAGE_MODES.INDEX">
-        <img ng-src="<[$ctrl.bannerImageFileUrl]>" align="center"
-             width="100%" alt="">
+        <img ng-src="<[$ctrl.bannerImageFileUrl]>" align="center"width="100%" alt="">
```

Comment:
@jimbyo shouldn't there be an alt text here for accessibility?

Reply:
Yup, please add alt here and below.

Reply:
Done.